runtime/doc/vim_diff.txt

```diff
@@ -126,7 +126,6 @@ Commands:
 Functions:
   |dictwatcheradd()| notifies a callback whenever a |Dict| is modified
   |dictwatcherdel()|
-  |execute()| works with |:redir|
```

Comment:
Vim has `execute()`, so it can’t be in vim_diff any longer. And in any case implementation has nothing to do with `:redir`.